partiql-ast/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+plugins {
+    id(Plugins.conventions)
+    id(Plugins.publish)
+    id(Plugins.pig)
+}
+
+dependencies {
+    api(Deps.pigRuntime)
+    api(Deps.ionElement)
+}
+
+pig {
+    namespace = "org.partiql.ast.domains"
```

Comment:
I think we should keep the package as `org.partiql.lang.domains` that way this is not a breaking change. The goal is to cleanly split artifacts of partiql-lang-kotlin for independent consumption. Ideally doing so without repackaging (ie breaking things) where possible.